Simulation/ThrustModel.m

```diff
@@ -0,0 +1,2 @@
+%Thrust Model

%MaxThrust = 100;
%Throttle = 1;
%airspeed = 50;

%Thrust = MaxThrust*Throttle*(1-airspeed/50)

function [Thrust_x, Thrust_y] = ThrustModel(StateVector)
  airspeed = sqrt(StateVector(3)^2 + StateVector(4)^2);
  Throttle = StateVector(6);
  MaxThrust = 100;
  Pitch = deg2rad(StateVector(5));
  Thrust = MaxThrust*Throttle*(1-airspeed/50);
  Thrust_y = Thrust*sin(Pitch);
  Thrust_x = Thrust*cos(Pitch);

```

Comment:
Here are the actions:

1. Looks like something messed up with the line ending type. Make sure you are using LF line ending type (Unix)
2. Remove the comments from the beginning (besides the first line) and add an overview of the function and any sources you may have used to write this (look at the AtmosphereModel.m for an example)
3. Change the output to one variable but as an array like so:
`Thrust = [Thrust_x, Thrust_y];`
This will let us do simple addition and subtraction on arrays
4. Since we are using degrees, use the `sind()` and `cosd()` functions instead of `sin()` and `cos()`.